pytests/test_wff_family_tax_credit.py

```diff
@@ -0,0 +1,133 @@
+from . import Reasoner
+
+
+class TestWFFTaxCreditKey(Reasoner):
+
+    key = 'isWorkingForFamiliesFamilyTaxCredit'
+
+
+class TestWFFTaxCreditDefault(TestWFFTaxCreditKey):
+
+    """
+    Benefit: Working for Families - Family Tax Credit (default state):
+    Default benefit.isWorkingForFamiliesFamilyTaxCredit is FORBIDDEN
+    Overriden by: Benefit: Working for Families
+     - Family Tax Credit (eligibility)
+    """
+
+    body = {}
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_forbidden)
+        self.assertTrue(self.is_conclusive)
+
+
+class TestWFFTaxCredit(TestWFFTaxCreditKey):
+
+    """
+    Benefit: Working for Families - Family Tax Credit (eligibility):
+        If applicant.isParent
+            and 20 ≤ applicant.worksWeeklyHours
+            and not applicant.receivesIncomeTestedBenefit
+            and not benefit.isStudentAllowance
+            and applicant.isPrincipalCarer
+            and not benefit.isOrphansBenefit
+                then benefit.isWorkingForFamiliesFamilyTaxCredit is PERMITTED
+    """
+
+    body = {
+        "applicant": {
+            "isParent": True,
+            "worksWeeklyHours": 20,
+            "receivesIncomeTestedBenefit": False,
+            "isPrincipalCarer": True,
+            "isPrincipalCarerForOneYearFromApplicationDate": False,
+            "Age": 16,
+            "isNZResident": True,
+            "normallyLivesInNZ": True
+        },
+        "benefit": {
+            "isStudentAllowance": False,
+            "isOrphansBenefit": False
+        },
+        "parents": {
+            "areDeceasedMissingOrIncapableThroughDisability": False
+        },
+        "child": {
+            "isDependent": False
```

Comment:
Just learned we're gonna need to change this to a new atom: `child.isFinanciallyIndependent` , which is different to being dependent. 

A child is financially dependent (but still dependent) if they are working 30 hours a week, getting student allowance, or on their own benefit/other govt assistance

Reply:
@Br3nda Thanks I will mention this in the meeting today.